src/app/products/manage-tax-components/view-tax-component/view-tax-component.component.html

```diff
@@ -0,0 +1,82 @@
+<div class="container m-b-20" fxLayout="row" fxLayout.lt-md="column" fxLayoutAlign="end" fxLayoutGap="2%">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+</div>
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" class="content">
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Name
+        </div>
+
+        <div fxFlex="50%">
+          {{ taxComponentData.name }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Percentage
+        </div>
+
+        <div fxFlex="50%">
+          {{ taxComponentData.percentage }}
+        </div>
+
+        <div *ngIf="taxComponentData.debitAccountType">
+          <div fxFlex="50%" class="mat-body-strong">
```

Comment:
```suggestion
          <div fxFlex="50%" class="mat-body-strong" *ngIf="taxComponentData.debitAccountType">
```
Add the condition separately to the required inner divs, don't create a separate outer div. Make changes at all places accordingly.